### ISSUE TITLE
Multiple 522's for fake avatar images from fakercloud

### ISSUE THREAD

**Issue by AndrewCathcart** [Marak/faker.js#1253]
E.g. https://cdn.fakercloud.com/avatars/codysanfilippo_128.jpg
https://cdn.fakercloud.com/avatars/ricburton_128.jpg

**Comment by MilosPaunovic** [comment]
Those links are loaded properly in my browser; it could just be a hiccup somewhere and you were caught in the wrong place at the wrong time.

**Comment by AndrewCathcart** [comment]
That's strange. Still no luck on my end, even on my mobile network it seems like the links don't work. Must be some routing issue to my area? Based in the UK.

**Comment by codeOfTheFuture** [comment]
I'm having the same issue.

**Comment by conradsze** [comment]
I'm having the same issue too

**Comment by cyrusonline** [comment]
a week ago no problem, but this week all or some image are broken.

**Comment by cyrusonline** [comment]
> Those links are loaded properly in my browser; it could just be a hiccup somewhere and you were caught in the wrong place at the wrong time.

which browser you are using??

**Comment by cyrusonline** [comment]
> That's strange. Still no luck on my end, even on my mobile network it seems like the links don't work. Must be some routing issue to my area? Based in the UK.

are you working on some JavaScript framework? Which are you working on ? Angular, vue , react or next?

**Comment by Eljoy** [comment]
same

**Comment by ndth4ng** [comment]
same

**Comment by AndrewCathcart** [comment]
> > That's strange. Still no luck on my end, even on my mobile network it seems like the links don't work. Must be some routing issue to my area? Based in the UK.
> 
> are you working on some JavaScript framework? Which are you working on ? Angular, vue , react or next?

I'm using faker in a Next.js project, but I don't think that's relevant at all. Trying to visit the images directly in my browser still results in an error. 

Still having this issue as of today as well.

**Comment by FourLineCode** [comment]
i have the same issue .. the cdn is returning multiple 522 errors .. i was loading about 30-50 images and about 2-3 of them succeed .. the rest are returning 522 .. 
its been 3-4 days since its happening for me

**Comment by liam1703** [comment]
I am having the same issue using within a VIte (Vue) project

**Comment by MilosPaunovic** [comment]
There is definitely something wrong with the entire [https://fakercloud.com/](https://fakercloud.com/), it isn't loading at all.
@Marak Could you confirm whether this is something planned or is it an unexpected error?

**Comment by AndrewCathcart** [comment]
Not sure if this is related to maraks pinned issue "pay me or fork this". The last comment he left several days ago was an ominous "execute order 66" and then he closed the issue.

**Comment by jrenschler** [comment]
Definitely seeing the same thing via direct URL and via application

**Comment by ekinertac** [comment]
having the same issue more than a week

**Comment by bhatvikrant** [comment]
Facing the same issue :(

**Comment by B3Kay** [comment]
Same...

**Comment by jeremyzilar** [comment]
Yes, getting 522 errors for all Fakercloud images, regardless if I load them in my app or in the browser.
Example: https://cdn.fakercloud.com/avatars/ricburton_128.jpg

![image](https://user-images.githubusercontent.com/395641/144665896-cdd90efb-5451-4b9a-a1f6-25de3b91b174.png)

**Comment by mattwhitson** [comment]
Same problem with my Nextjs project, hopefully someone is able to fix the issue soon :)

**Comment by ParogDev** [comment]
> Not sure if this is related to maraks pinned issue "pay me or fork this". The last comment he left several days ago was an ominous "execute order 66" and then he closed the issue.

Developers get fed up, maintaining a package can take a lot of time and there's a lot of ungrateful people that take projects like these for granted. I only just discovered it but it seems @Marak (Don't mind the tag. :) ) has stopped maintaining it. 

I believe the host for the images was @Marak 's and it only makes sense to stop giving this service for free as it does come with a cost to him. From what I gathered he is stepping away from the project and I think we should respect that. 

As developers however, instead of forking to many different updates of this, how can we organize ourselves for a community maintained version of this? Maybe this question would fit better as a separate issue, but I felt this information should be here  for people to understand that avatars most likely won't work the way they used to without an update from another dev, using a different host / service.

Wish you the best of luck and happiness, @Marak. :)

**Comment by renanmonsi** [comment]
I'm having the same problem. (I'm just commenting here to get up).

**Comment by nicolasbarcellos** [comment]
Same here

**Comment by chinmay-prajapat** [comment]
Me too facing the same.

**Comment by ibrahimzahir** [comment]
broken

**Comment by kan1657** [comment]
Same problem with my Nextjs project
